main/mafia.hs

```diff
@@ -750,7 +750,7 @@ appendStandardDirs dir =
 
 standardSourceDirs :: [Directory]
 standardSourceDirs =
-  ["src", "test", "gen", "dist/build", "dist/build/autogen"]
+  ["src", "test", "gen", "dist/build", "dist/build/autogen", "dist/build/global-autogen"]
```

Comment:
Is that something new in cabal 2.0 ?

Reply:
New in cabal 2.0 and GHC 8.2 apparently.